packages/common/src/SolcoverConfig.js

```diff
@@ -8,7 +8,8 @@ const SolcoverConfig = {
     "common/test",
     "oracle/test",
     "oracle/implementation/test"
-  ]
+  ],
+  measureStatementCoverage: false
```

Comment:
Can you explain this change?

Reply:
This is the change i referred in the call. When you add number of tests and their volume, there is a bug in solidity coverage that returns memory-out-of-bound ( See https://github.com/sc-forks/solidity-coverage/issues/543). This is the fix according to the suggestion of the issue page

Reply:
I see. So this is turning off a feature to make it more memory-efficient?

Reply:
Yes, setting this option to false we increase memory performance of solidity coverage and tests don't fail.